blackbox/docs/protocols/postgres.txt

```diff
@@ -142,58 +135,54 @@ implemented::
+  As a workaround it's possible to use ``SHOW TABLES`` or query
+  the ``information_schema`` tables manually using ``SELECT`` statements.
 
-- ``OBJECT`` and ``GEO_SHAPE`` can be streamed as ``JSON`` but require at least
-  version ``9.4.1210`` of `pgjdbc`_.
+- ``OBJECT`` and ``GEO_SHAPE`` columns can be streamed as ``JSON`` but require
+  `pgjdbc`_ version ``9.4.1210`` or newer.
 
-- Multidimensional arrays will be streamed as ``JSON`` encoded string. This is
-  done to avoid a limitation of the postgres binary streaming protocol in which
-  all sub-arrays are required to have the same length.
+- Multidimensional arrays will be streamed as ``JSON`` encoded string to avoid
+  a protocol limitation where all sub-arrays are required to have the same length.
 
-- The behaviour of ``PreparedStatement.executeBatch`` in error cases depends on
-  in which stage an error occurs.
-  It may raise ``BatchUpdateException`` without processing any items.
-  Or it may continue processing the items in which case the result might
-  contain individual items marked with ``EXECUTE_FAILED`` (-3).
+- The behavior of ``PreparedStatement.executeBatch`` in error cases depends on
+  in which stage an error occurs: A ``BatchUpdateException`` is thrown if no
+  processing has been done yet, whereas single operations failing after the
+  processing started are indicated by an ``EXECUTE_FAILED`` (-3) return value.
 
-- Transaction limitations as described in the beginning of this document apply.
+- Transaction limitations as described above.
 
-- Enabled escape processing may prevent the usage of :ref:`Object
-  Literals <data-type-object-literals>` when an object key starting
+- Having ``escape processing`` enabled could prevent the usage of :ref:`Object
+  Literals <data-type-object-literals>` in case an object key's starting
   character clashes with a JDBC escape keyword (see also `JDBC escape syntax
   <https://docs.oracle.com/javadb/10.10.1.2/ref/rrefjdbc1020262.html>`_).
-  Disabling escape processing is currently the only solution, but when
-  using the `Extended Query`_ API this does not work due to a `bug
-  <https://github.com/pgjdbc/pgjdbc/issues/653>`_ at `pgjdbc`_ which is
-  currently only solved by an unreleased snapshot (`9.4.1212.jre7-SNAPSHOT`).
+  Currently, disabling ``escape processing`` will remedy this, but prevent
+  the `Extended Query`_ API from working due to a `bug
+  <https://github.com/pgjdbc/pgjdbc/issues/653>`_ at `pgjdbc`_.
 
 
-Connection failover and load balancing
+Connection Failover and Load Balancing
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Connection failover and load balancing is supported as described here:
 `PostgreSQL JDBC connection failover`_.
 
 .. note::
 
-    **targetServerType** parameter shouldn't be used as for Crate there is no distinction between
-    master and slave nodes.
+    It is not recommended to use the **targetServerType** parameter since Crate
+    has no Master/Slave nodes.
```

Comment:
Would rephraze:
... doesn't have the concept of Master/Slave nodes
or
... in Crate there is no distinction of the nodes into master/slave